src/App.jsx

```diff
@@ -1,33 +1,63 @@
+import { useState } from 'react';
+
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-export const App = () => (
-  <div className="page">
-    <div className="page-content">
-      <div className="box">
-        <div className="field">
-          {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
-          <label htmlFor="search-query" className="label">
-            Search movie
-          </label>
-
-          <div className="control">
-            <input
-              type="text"
-              id="search-query"
-              className="input"
-              placeholder="Type search word"
-            />
+const changeToLowerCase = string => string.toLowerCase();
+const removeSpaca = query => query.toLowerCase().trim();
+
+function preperedMoviesFromServer(movies, { query }) {
```

Comment:
```suggestion
function getPreperedMoviesFromServer(movies, { query }) {
```
Name functions as verbs.